### PR TITLE
cast int metrics to float. avoid influx db cast exception

### PR DESCRIPTION
**Pull request by tmaitz**

### CODE REVIEW

**Comment by tmaitz** [comment]
fix https://github.com/yandex/yandex-tank/issues/776

File: yandextank/plugins/InfluxUploader/decoder.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def decode_monitoring(self, data):
                         {"host": host, "comment": host_data.get("comment")},
                         second_data["timestamp"],
                         {
-                            metric: value
+                            # cast int to float. avoid https://github.com/yandex/yandex-tank/issues/776
+                            metric: float(value) if type(value) is int else value
```
Comment (eIGato) [inline yandextank/plugins/InfluxUploader/decoder.py]: metric: float(value) if isinstance(value, int) else value

Comment (tmaitz) [reply yandextank/plugins/InfluxUploader/decoder.py]: done

**Comment by tmaitz** [comment]
> Please add a test for this case.

test added

**Comment by eIGato** [comment]
My remaining comments are just suggestions for cosmetic changes.

**Pull request merged**